src/lib/merkle_ledger_tests/visualizable_ledger.ml

```diff
@@ -1,46 +1,143 @@
 open Core
+open Async
 
-module type Ledger_intf = sig
-  type t
+(** Visualizable_ledger shows a subgraph of a merkle_ledger using Graphviz *)
+module type S = sig
+  type addr
 
-  type hash [@@deriving sexp, eq]
+  type tree
 
-  val max_depth : int
+  type t
 
-  val get_inner_hash_at_addr_exn : t -> Direction.t list -> hash
+  (* Visualize will enumerate through all edges of a subtree with a root
```

Comment:
the word "root" is confusing here, because the initial address may be other than the tree root; can probably just omit it